server/scripts/import-tests/index.js

```diff
@@ -53,7 +53,8 @@ Default use:
 const client = new Client();
 
 const ariaAtRepo = 'https://github.com/w3c/aria-at.git';
-const ariaAtDefaultBranch = 'master';
+// const ariaAtDefaultBranch = 'master';
```

Comment:
This needs to be reverted before merging.

Reply:
Nice trick!